app/controllers/concerns/potential_spam_user.rb

```diff
@@ -0,0 +1,21 @@
+# This module is the source of truth for what we consider a potential spam user.
+module PotentialSpamUser
+  extend ActiveSupport::Concern
```

Comment:
How about making this a concern and including it, we just turn this into an object? Something like

```ruby
@potential_spam_users = PotentialSpamUsers.call 
```

in the controller seems easy enough, WDYT?